genomics_quality_control/genomics_quality_control.md

```diff
@@ -0,0 +1,927 @@
+
+```bash
+ls
+```
+
+``` +output
+SRR2584863_1.fastq        SRR2584866_1_fastqc.html  SRR2589044_1_fastqc.html
+SRR2584863_1_fastqc.html  SRR2584866_1_fastqc.zip   SRR2589044_1_fastqc.zip
+SRR2584863_1_fastqc.zip   SRR2584866_1.fastq.gz     SRR2589044_1.fastq.gz
+SRR2584863_2_fastqc.html  SRR2584866_2_fastqc.html  SRR2589044_2_fastqc.html
+SRR2584863_2_fastqc.zip   SRR2584866_2_fastqc.zip   SRR2589044_2_fastqc.zip
+SRR2584863_2.fastq.gz     SRR2584866_2.fastq.gz     SRR2589044_2.fastq.gz
+```
+For each input FASTQ file, FastQC has created a `.zip` file and a
+`.html` file.
+The `.zip` file extension indicates that this is actually a compressed set of multiple output files.
+We'll be working with these output files soon.
+The `.html` file is a stable webpage displaying the summary report for each of our samples.
+
+We want to keep our data files and our results files separate,
+so we will move these output files into a new directory within our `results/` directory.
+
+```bash
+mkdir -p ~/genomics_tools_and_methods/results/fastqc_untrimmed_reads
+mv *.zip ~/genomics_tools_and_methods/results/fastqc_untrimmed_reads/
+mv *.html ~/genomics_tools_and_methods/results/fastqc_untrimmed_reads/
+```
+
+Now we can navigate into this results directory and do some closer inspection of our output files.
+
+```bash
+cd ~/genomics_tools_and_methods/results/fastqc_untrimmed_reads/
+```
+
+### Viewing the FastQC results
+
+If we were working on our local computers, we would be able to look at each of these HTML files by opening them in a web browser.
+
+However, these files are currently sitting on our remote AWS instance, where our local computer can not see them. And, since we are only logging into the AWS instance via the command line - it does not have any web browser setup to display these files either.
+
+So the easiest way to look at these webpage summary reports will be to transfer them to our local computers (i.e. your laptop).
+
+To transfer a file from a remote server to our own machines, we will use `scp`.
+
+<div class = "learn-more">
+<b style="color: rgb(var(--color-highlight));">Learning connection</b><br>
+
+For a review of `scp`, see the section on [transferring data between your local machine and the cloud](https://datacarpentry.org/shell-genomics/05-writing-scripts/#transferring-data-between-your-local-machine-and-the-cloud) from the Data Carpentry Shell Genomics lesson.
+
+</div>
+
+First we will make a new directory on our computer to store the HTML files we are transferring.
+Let’s put it on our desktop for now.
+Open a new tab in your terminal program (you can use the pull down menu at the top of your screen or the Cmd+t keyboard shortcut) and type:
+
+```bash
+scp dcuser@ec2-34-238-162-94.compute-1.amazonaws.com:~/genomics_tools_and_methods/results/fastqc_untrimmed_reads/*.html ~/Desktop/fastqc_html
+```
+
+As a reminder, the first part
+of the command `dcuser@ec2-34-238-162-94.compute-1.amazonaws.com` is
```

Comment:
I'd consider moving this reminder up to before the scp code instead of after, to maybe help prevent folks quickly copying and pasting and forgetting to change the instance number (I totally made this mistake myself!)

Reply:
Good call! Will do.